components/hab/win/win-build.ps1

```diff
@@ -103,12 +103,38 @@ $env:OPENSSL_LIB_DIR            = $ChocolateyHabitatLibDir
 $env:OPENSSL_INCLUDE_DIR        = $ChocolateyHabitatIncludeDir
 $env:OPENSSL_STATIC             = $true
 
-
-# Start the build
 if (-not (Test-AppVeyor)) {
+    # Start the build
     Push-Location "$psscriptroot\.."
     invoke-expression "$cargo clean"
-    Invoke-Expression "$cargo build" 
+    Invoke-Expression "$cargo build --release" 
     Pop-Location
+
+    # Create the archive
+    $pkgRoot = "results"
+    New-Item -ItemType Directory -Path $pkgRoot -ErrorAction SilentlyContinue
+
+    $pkgRelease = (Get-Date).ToString('yyyyMMddhhmmss')
+    $pkgVersion = (Get-Content -Path ..\..\..\VERSION | Out-String).Trim()
+    $pkgArtifact = "$pkgRoot/core-hab-$pkgVersion-$pkgRelease-x86_64-windows"
+    Compress-Archive -Path @(
```

Comment:
In order for this to be a valid .hart file, we'll probably need to make it a tar.xz and include the appropriate metadata files.  Otherwise, `hab install` will error (which is part of the process for the publish-hab process).